# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1351490572 -3600
#      Mon Oct 29 07:02:52 2012 +0100
# Node ID 204cafff778c6540674865b8a7646c53a84db927
# Parent  ab1c6e6d1be6d9f6ed75c473718514578f508766
terminal window with octave logo as icon when floating

* main-window.cc: do not set terminal icon in construct ()

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -491,17 +491,16 @@ main_window::construct ()
   _current_directory_up_tool_button = new QToolButton (this);
   _current_directory_up_tool_button->setIcon (QIcon(":/actions/icons/up.png"));
 
   // Octave Terminal subwindow.
   _terminal = new QTerminal (this);
   _terminal->setObjectName ("OctaveTerminal");
   _terminal->setFocusPolicy (Qt::StrongFocus);
   _terminal_dock_widget = new terminal_dock_widget (_terminal, this);
-  _terminal_dock_widget->setWindowIcon (QIcon(":/actions/icons/terminal.png"));
 
   QWidget *dummyWidget = new QWidget ();
   dummyWidget->setObjectName ("CentralDummyWidget");
   dummyWidget->resize (10, 10);
   dummyWidget->setSizePolicy (QSizePolicy::Minimum, QSizePolicy::Minimum);
   dummyWidget->hide ();
   setCentralWidget (dummyWidget);
 
